etc/cp-ksql-server/_base_observations_stream.sql

```diff
@@ -0,0 +1,20 @@
+SET 'auto.offset.reset' = 'earliest';
```

Comment:
How are the sql files being loaded to the KSQL server? Right now it only accepts the queries as a [text config](https://github.com/RADAR-base/cp-helm-charts/blob/9b9f897be4dba89102cd92ba6e8a33fbd56ed2c0/charts/cp-ksql-server/values.yaml#L87).

Reply:
@yatharthranjan Please check the readme of this PR. It is done via _base.yaml.gotmpl_.

Reply:
ok thanks, i missed that.